pkgs/by-name/po/porn-vault/porn-vault.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+cd $(mktemp -d)
+for i in %out%/share/porn-vault/*; do
+  ln -s $i .
+done
+[[ -z "${PV_CONFIG_FOLDER}" ]] || install -Dm111 %izzy%/bin/izzy $PV_CONFIG_FOLDER/izzy
+exec %nodejs%/bin/node ./dist/index.js
```

Comment:
```suggestion
exec @nodejs@/bin/node ./dist/index.js
```
We need to use the syntax of substituteAll, though